proto/rpc/user/group.pb.gw.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/utilities"
 	"github.com/paralus/paralus/proto/types/commonpb/v3"
-	"github.com/paralus/paralus/proto/types/userpb/v3"
+	userv3_0 "github.com/paralus/paralus/proto/types/userpb/v3"
```

Comment:
looks like these are due to a different plugin version, let's avoid such changes as part of this feature. applicable to all other *pb files that are irrelavent to this feature.

Reply:
proto version issue, unresolved